src/collectors/windows.plugin/metadata.yaml

```diff
@@ -1206,6 +1206,92 @@ modules:
               chart_type: line
               dimensions:
                 - name: uptime
+        - name: IIS W3SCV W3MP
+          description: "These metrics refer to the World Wide Web Publishing Service, the service responsible for hosting and serving web content"
+          labels:
+            - name: app
+              description: The name of the Application running.
+            - name: pid
+              description: Current PID of the process.
+          metrics:
+            - name: iis.w3scv_w3wp_active_threads
+              description: Threads actively processing requests in the worker process.
+              unit: "threads"
+              chart_type: line
+              dimensions:
+                - name: threads
+            - name: iis.w3scv_w3wp_requests_total
+              description: Maximum number of threads to which the thread pool can grow as needed.
+              unit: "requests"
+              chart_type: line
+              dimensions:
+                - name: requests
+            - name: iis.w3scv_w3wp_requests_active
+              description: Current number of requests being processed by the worker process.
+              unit: "requests"
+              chart_type: line
+              dimensions:
+                - name: requests
+            - name: iis.w3scv_w3wp_file_cache_mem_usage
+              description: Current memory usage by the worker process.
+              unit: "bytes/s"
+              chart_type: line
+              dimensions:
+                - name: used
+            - name: iis.w3scv_w3wp_files_cache_total
+              description: Total number of files whose contents were ever added to the cache.
+              unit: "files/s"
+              chart_type: line
+              dimensions:
+                - name: files
+            - name: iis.w3scv_w3wp_files_flushed_total
+              description: Total number of file handles that have been removed from the cache.
+              unit: "files/s"
+              chart_type: line
+              dimensions:
+                - name: files
+            - name: iis.w3scv_w3wp_uri_cache_flushed
+              description: Total number of URI cache flushes.
+              unit: "flushes/s"
+              chart_type: line
+              dimensions:
+                - name: flushes
+            - name: iis.w3scv_w3wp_total_uri_cached
+              description: Total number of URI information blocks added to the cache.
+              unit: "blocks/s"
+              chart_type: line
+              dimensions:
+                - name: blocks
+            - name: iis.w3scv_w3wp_total_metadata_cache
+              description: Number of metadata information blocks currently present in user-mode cache.
+              unit: "blocks/s"
+              chart_type: line
+              dimensions:
+                - name: blocks
```

Comment:
Why "blocks/s" if this is blocks currently present?

Reply:
For this specific case, I neglected to update both the data collection logic and metadata. The official description and implemented algorithms were unclear.